src/Avalonia.Dialogs/Avalonia.Dialogs.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AvaloniaResource Include="**\*.xaml">
+      <SubType>Designer</SubType>
+    </AvaloniaResource>
+  </ItemGroup>
+
+  <Import Project="..\..\build\Rx.props" />
```

Comment:
I'd rather not have any core parts of the framework depending on RXUI

Reply:
Yeah, we used to depend on RxUI in avalonia core, but had to remove it because they break compatibility really often.